solr/contrib/ltr/src/java/org/apache/solr/ltr/LTRScoringQuery.java

```diff
@@ -102,6 +102,26 @@ public LTRScoringModel getScoringModel() {
     return ltrScoringModel;
   }
 
+  public void setLtrScoringModel(LTRScoringModel ltrScoringModel) {
+    this.ltrScoringModel = ltrScoringModel;
+  }
+
+  public void setLtrThreadMgr(LTRThreadModule ltrThreadMgr) {
+    this.ltrThreadMgr = ltrThreadMgr;
+  }
+
+  public void setExtractAllFeatures(boolean extractAllFeatures) {
+    this.extractAllFeatures = extractAllFeatures;
+  }
+
+  public void setEfi(Map<String, String[]> efi) {
+    this.efi = efi;
+  }
```

Comment:
LTRFeatureLoggerTransformerFactory.8 - with a `rerankedQueriesFromContext` + `rerankedQueries` approach in the feature logger transformer the setters here would not be needed.